docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     image: ollama/ollama
     container_name: ollama
     ports:
-      - "11434:11434"
+      - "5000:5000"
```

Comment:
not correct, Ollama exposes the models on 11434

Reply:
fixed